### PR TITLE
Applications: nrf5340_audio: Updated controller

### PR DESCRIPTION
**Pull request by koffes**
OCT-2840
LE Audio controller subsystem updated to 17933

### CODE REVIEW

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,16 @@ Changelog
...
 
+Controller v17933
+*****************
+
+There will soon be a full transition from this controller to the SoftDevice Controller.
+The following changes have been introduced in the v17933 of the controller:
+
+Changes
+=======
+* Updated to be based on ...
```
Comment (koffes) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:20]: Changes to be added.

**Review by rick1082 — APPROVED** [review]

**Review by alexsven — APPROVED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,16 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v17933
+*****************
+
+There will soon be a full transition from this controller to the SoftDevice Controller.
+The following changes have been introduced in the v17933 of the controller:
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:16]:
```suggestion
.. note::
     This controller is now :ref:`deprecated <api_deprecation>`. Before it is removed, make sure to transition to using :ref:`softdevice_controller`.

The following changes have been introduced in the v17933 of the controller:
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Would be good to add transition steps in the migration guide and a link to them, but there are three different PRs now addressing the same topic, so it is complicated to do this...

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 3]
```diff
@@ -9,6 +9,24 @@ Changelog
...
+Changes
+=======
+* Improved the robustness for CIS bidirectional four streams (two sinks and two sources) scenario.
+* Improeve the stability for switching between BIS and CIS scenario in runtime.
+* Corrected the timestamps for BIS.
+* Fixed the wrong LLID from ISO empty packets.
+* Corrected the disconnection reason for CIS.
+* Bugfixes and stability Improvements.
+* PTO not supported; sink will accept PTO=0 or PTO=1.
+* Optimize CIS setup time.
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:28]:
```suggestion
.. note::
     This controller is now :ref:`deprecated <api_deprecation>`. Before it is removed, make sure to transition to using :ref:`softdevice_controller`.
 
The following changes have been introduced in the v17933 of the controller:

Changes
=======

* Improved the robustness for the CIS bidirectional four stream scenario (two sinks and two sources).
* Improved the stability for switching between the BIS and the CIS scenarios at runtime.
* Corrected the timestamps for BIS.
* Fixed the wrong Link Layer ID (LLID) from ISO empty packets.
* Corrected the disconnection reason for CIS.
* Modified handling of the missing support for pre-transmission offset (PTO). The sink will now accept ``PTO=0`` or ``PTO=1`` values and will not fail in either case.
* Optimized the CIS setup time.
* Bugfixes and stability improvements.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Please apply the blank lines after the note line and after the header.

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 4]
```diff
@@ -9,6 +9,24 @@ Changelog
...
+Controller v17933
+*****************
+
+.. note::
+     This controller is now :ref:`deprecated <api_deprecation>`. Before it is removed, make sure to transition to using :ref:`softdevice_controller`.
+
+The following changes have been introduced in the v17933 of the controller:
+Changes
+=======
+* Improved the robustness for the CIS bidirectional four stream scenario (two sinks and two sources).
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:21]:
```suggestion
The following changes have been introduced in the v17933 of the controller:

Changes
=======

* Improved the robustness for the CIS bidirectional four stream scenario (two sinks and two sources).
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**